source/content/php-versions.md

```diff
@@ -36,11 +36,11 @@ Changes made to the `pantheon.yml` file on a branch **are not** detected when cr
 | [7.4](https://v74-php-info.pantheonsite.io/) | ✅          | Active  |
 | [7.3](https://v73-php-info.pantheonsite.io/) | ❌          | EOL     |
 | [7.2](https://v72-php-info.pantheonsite.io/) | ❌          | EOL     |
-| [7.1](https://v71-php-info.pantheonsite.io/) | ❌          | EOL     |
-| [7.0](https://v70-php-info.pantheonsite.io/) | ❌          | EOL     |
+| [7.1](https://v71-php-info.pantheonsite.io/) | 🛑          | EOL     |
+| [7.0](https://v70-php-info.pantheonsite.io/) | 🛑          | EOL     |
 | [5.6](https://v56-php-info.pantheonsite.io/) | ❌          | EOL |
-| [5.5](https://v55-php-info.pantheonsite.io/) | ❌          | EOL |
-| [5.3](https://v53-php-info.pantheonsite.io/) | ❌          | EOL * |
+| [5.5](https://v55-php-info.pantheonsite.io/) | ❌❗️          | End-of-Sale <Popover title="End-of-Sale" content="End-of-Sale versions are no longer available to new sites on the platform.  Existing sites using these versions will be automatically upgraded in the future." /> |
+| [5.3](https://v53-php-info.pantheonsite.io/) | 🚫          | Retired <Popover title="Retired" content="End-of-Sale versions are no longer available to new sites on the platform.  Existing sites using these versions will be automatically upgraded in the future. To resume development on a site using a retired version of PHP, upgrade the PHP version on the development environment." />* |
```

Comment:
I don't think that we need to differentiate 5.3 from 5.5; even if 5.5 is more functional than 5.3, we could still say that we don't support doing new development with any end-of-sale version of PHP.

Reply:
Agree... when I put the 'retired' there, it was to show what it might look like after a version was no longer available.. but I agree that it's confusing in this PR, even though the PR is just "brainstorming"